lib/vicadmin/validate.go

```diff
@@ -62,6 +63,47 @@ const (
 	BadStatus  = template.HTML(`<i class="icon-attention"></i>`)
 )
 
+func GetMgmtIP() net.IPNet {
+	var mgmtIP net.IPNet
+	// management alias may not be present, try others if not found
+	link := LinkByNameOrAlias("management")
```

Comment:
after the code example below this would collapse to:
LinkByOneOfNameOrAlias("management", "external", "client")